Text/Shakespeare/Base.hs

```diff
@@ -21,6 +21,9 @@ module Text.Shakespeare.Base
     , derefToExp
     , flattenDeref
     , readUtf8File
+    , readFileUtf8
```

Comment:
Looks good. Just one last set of changes. These new identifiers will be part of the new API. Please:

* Add a minor version bump to the cabal file
* Add an explanation of the changes and new API to the ChangeLog
* Add Haddock comments including `@since` comments to the identifiers